README.md

```diff
@@ -2,423 +2,85 @@
 
 # pino&nbsp;&nbsp;[![Build Status](https://travis-ci.org/pinojs/pino.svg?branch=master)](https://travis-ci.org/pinojs/pino)&nbsp;[![Coverage Status](https://coveralls.io/repos/github/pinojs/pino/badge.svg?branch=master)](https://coveralls.io/github/pinojs/pino?branch=master) [![TypeScript definitions on DefinitelyTyped](http://definitelytyped.org/badges/standard.svg)](http://definitelytyped.org)
 
-[Extremely fast](#benchmarks) node.js logger, inspired by Bunyan.
-
-* [Installation](#install)
-* [Usage](#usage)
-* [Benchmarks](#benchmarks)
-* [API ⇗](docs/api.md)
-* [FAQ ⇗](docs/faq.md)
-* [Redaction ⇗](docs/redaction.md)
-* [Ecosystem ⇗](docs/ecosystem.md)
-* [Pretty Printing ⇗](docs/pretty.md)
-* [Extreme mode explained ⇗](docs/extreme.md)
-* [Transports with Pino](#transports)
-* [Pino in the browser](#browser)
-* [Caveats](#caveats)
-* [Team](#team)
-* [Contributing](#contributing)
-* [Acknowledgements](#acknowledgements)
-* [License](#license)
+[Very low overhead](#benchmarks) Node.js logger, inspired by Bunyan.
+
+## Documentation
+
+* [Benchmarks ⇗](/docs/benchmarks.md)
+* [API ⇗](/docs/api.md)
+* [Browser API ⇗](/docs/browser.md)
+* [Redaction ⇗](/docs/redaction.md)
+* [Child Loggers ⇗](/docs/child-loggers.md)
+* [Transports ⇗](/docs/transports.md)
+* [Web Frameworks ⇗](/docs/web.md)
+* [Pretty Printing ⇗](/docs/pretty.md)
+* [Extreme Mode ⇗](/docs/extreme.md)
+* [Ecosystem ⇗](/docs/ecosystem.md)
+* [Help ⇗](/docs/help.md)
 
 ## Install
 
 ```
-npm install pino --save
+$ npm install pino --save
 ```
 
-If you need support for Node.js v0.12 or v0.10, please install the latest 2.x release using the `legacy` tag:
```

Comment:
We need to add a section on pino v4 and pino v2 somewhere.

Reply:
– todo: add docs/migration-and-legacy.md